### ISSUE TITLE
View.propTypes has been deprecated

### ISSUE THREAD

**Issue by rhysdavies1994** [kiok46/react-native-multi-toggle-switch#1]
First off, handy little library, nice work! 

Although today when building to release today, I got the following errors from both iOS and Android logging:
```
...
ReactNativeJS: undefined is not an object (evaluating 'l.View.propTypes.style')
ReactNativeJS: Module AppRegistry is not a registered callable module (calling runApplication)
ReactNative: ReactInstanceManager.detachViewFromInstance()
ReactNativeJS: Module AppRegistry is not a registered callable module (calling unmountApplicationComponentAtRootTag)
```

I eventually tracked down the issue to this library using `View.propTypes.style`, but React Native has changed this recently to "ViewPropTypes".

**Comment by rhysdavies1994** [comment]
https://github.com/kiok46/react-native-multi-toggle-switch/pull/2

**Comment by rhysdavies1994** [comment]
I've made a PR with suggested fix: #2

**Comment by kiok46** [comment]
Merged #2
